Sources/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+#[[
+This source file is part of the DocC open source project
+
+Copyright (c) 2024 Apple Inc. and the DocC project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See https://swift.org/LICENSE.txt for license information
+#]]
+
+add_subdirectory(CLMDB)
```

Comment:
Since there's only the one subdirectory, we could consider omitting this file altogether and jumping straight into `Sources/CLMDB` from the top-level. Thoughts?

Reply:
It is generally suggested that you do the per-directory CMakeLists.txt. That does allow some level of flexibility. I mean, this is a small enough project that we could also just do it all inline in a single CMakeLists.txt.